recipes/psycopg2/build.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
+
+export LDFLAGS="${LDFLAGS} -L$PREFIX/lib -lssl"
+
+$PYTHON setup.py install
+
+if [[ `uname` == 'Darwin' ]]; then
+     PG_LIB=$(pg_config --libdir)
+     for LIBRARY in `find ${SP_DIR}/${PKG_NAME} -name "*.so"`;
+     do
+        install_name_tool -change libssl.1.0.0.dylib @rpath/libssl.1.0.0.dylib $LIBRARY
+        install_name_tool -change libcrypto.1.0.0.dylib @rpath/libcrypto.1.0.0.dylib $LIBRARY
```

Comment:
I'm assuming these files are part of a different package (otherwise conda build would just do this itself after the build is complete). Assuming that is correct, shouldn't you break the hard links here? It doesn't matter for conda-forge, but if someone tries to build this recipe on their own machine, this will write into the package cache.  Sorry if I'm missing something here.